src/corner_kick/scripts/start_corner_kick.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+# Start script for Corner Kick.
+
+# Get the current directory of the script.
+DIRECTORY=`dirname $0`
+
+# Move to this directory and start the production build of
+# Corner Kick.
+cd $DIRECTORY
+yarn prod
```

Comment:
nit: GitHub complaining about lack of newline at end of file.

Reply:
Fixed